misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -187,7 +187,7 @@ public List<String> skipList() {
                 "query_expr_source_116.bal", "query_expr_source_117.bal", "query_expr_source_118.bal",
                 "query_expr_source_119.bal", "query_expr_source_120.bal", "query_expr_source_121.bal",
                 "query_expr_source_122.bal", "query_expr_source_123.bal", "query_expr_source_124.bal",
-                "query_expr_source_103.bal", "query_expr_source_115.bal");
+                "query_expr_source_103.bal", "query_expr_source_115.bal", "query_expr_source_126.bal");
```

Comment:
Once the formatter changes are done we have to enable this. Let's add a TODO here